meta-mender-core/classes/mender-setup.bbclass

```diff
@@ -220,7 +220,11 @@ def mender_feature_is_enabled(feature, if_true, if_false, d):
     else:
         return if_false
 
-inherit ${@mender_feature_is_enabled('mender-client-install', 'mender-setup-distro-features', '', d)}
+# MENDER_FEATURES_ENABLE and MENDER_FEATURES_DISABLE map to
+# DISTRO_FEATURES_BACKFILL and DISTRO_FEATURES_BACKFILL_CONSIDERED,
+# respectively.
+DISTRO_FEATURES_BACKFILL_append = " ${MENDER_FEATURES_ENABLE}"
+DISTRO_FEATURES_BACKFILL_CONSIDERED_append = " ${MENDER_FEATURES_DISABLE}"
```

Comment:
From my commit message here:

> It is not necessary to check MENDER_FEATURES_ENABLE/DISABLE variables before setting the DISTRO_FEATURES variables, since both of these variables are empty unless you have included Mender features. Hence the whole operation is a no-op, which is what we want to be designated Yocto-compatible (layer doesn't change build OOTB).

The only problem I can envision is if Yocto requires *exactly* the same DISTRO_FEATURES string, because one space will be added here. But we can get around that by using a small snippet, like:

```
DISTRO_FEATURES_BACKFILL_append = "${@' ${MENDER_FEATURES_ENABLE}' if '${MENDER_FEATURES_ENABLE}' else ''}"
```